docs/v3/config.rb

```diff
@@ -15,21 +15,25 @@
 set :images_dir, 'images'
 set :fonts_dir, 'fonts'
 
+ignore 'includes/api_resources/examples/*.json'
```

Comment:
These json files are consumed with rendering templates, so don't need to be included in built resources.